test/common/README.md

```diff
@@ -642,6 +642,10 @@ environment variables.
 If set, `NODE_COMMON_PORT`'s value overrides the `common.PORT` default value of
 12346\.
 
+### `NODE_REGENERATE_SHANPSHOTS`
+
+If set, test snapshots are regenerated.
```

Comment:
Do you mind adding an example of using this? Does it regenerate _all_ snapshots? I think most of the time I’d only want to regenerate a particular one for the test I’m working on.